rest/model/host.go

```diff
@@ -87,9 +88,14 @@ func (apiHost *APIHost) buildFromHostStruct(h interface{}) error {
 	apiHost.Status = ToAPIString(v.Status)
 	apiHost.UserHost = v.UserHost
 
+	imageId, err := v.Distro.GetImageID()
+	if err != nil {
+		return fmt.Errorf("problem getting image ID")
```

Comment:
should wrap the error?

Reply:
I just followed the trend of this file and used `fmt.Errorf` (errors isn't even imported..). Can add import and wrap if you would like.

Reply:
should always just import things if we need it. fmt.Errorf isn't a thing we're calling very much/at all (it doesn't annotate the error with stack, and in this case it's not a format string, so you'd want to do errors.New anyway (if there weren't an error you wanted to wrap.)